clang/test/CheckedCRewriter/checkedptrret.c

```diff
@@ -0,0 +1,22 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+
+
+int* foo() {
+    int *c = 0;
+    return c;
+}
+
+// CHECK:   _Ptr<int> foo(void) _Checked{
```

Comment:
I am wondering: Should we format as `_Checked{` or `_Checked {` ? I see arguments both ways. Is there any reference in the Checked C codebase itself (e.g., examples) that would suggest one over the other?

Reply:
Fixed, space was added.